### PR TITLE
Add 2022-08-what-is-future-trading.md

### PR DESCRIPTION
**Pull request by Herklos**

### CODE REVIEW

File: 2022-08-what-is-future-trading.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+Cryptocurrency future trading is essentially betting on the future price of a particular coin. For example, let's say you think the price of Bitcoin is going to increase in the next month. You could buy a Bitcoin future contract that expires in one month and allows you to buy Bitcoin at the current price. If the price of Bitcoin does indeed increase in the next month, you would make a profit on your contract.
+
+There are a few reasons why cryptocurrency future trading can be a good option. First, it allows you to get exposure to the price movement of a particular coin without actually having to own any of the coins. This can be helpful if you don't want to tie up your capital in a particular coin but still want to benefit from its price movement.
+
+Another reason why cryptocurrency future trading can be attractive is that it often provides leverage. This means that you can control a larger position than if you were just buying the coins outright. This can amplify both
```
Comment (GuillaumeDSM) [inline 2022-08-what-is-future-trading.md]: This can amplify both ? I guess the end is missing :D

File: 2022-08-what-is-future-trading.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+6. Hedging: This strategy involves taking both long and short positions in different currencies to offset risk.
+
+# Automating future trading with OctoBot
+
+OctoBot is an open-source software project that automates cryptocurrency trading. It is designed to be easily extensible and adaptable, allowing it to be used with a wide range of exchanges and strategies.
+
+OctoBot has a number of features that make it well suited for future trading. First, it supports multiple exchanges, allowing you to trade on multiple platforms simultaneously. Second, it includes a risk management system that can automatically adjust your position size to limit your losses. Third, it features a number of built-in strategies that you can use or customize to suit your own trading style.
+
+Fourth, and perhaps most importantly, OctoBot is constantly being updated with new features and improvements. This means that it will continue to get better over time, making it an ideal tool for long-term future trading.
```
Comment (GuillaumeDSM) [inline 2022-08-what-is-future-trading.md]: **Finally**, and perhaps most importantly

**Review by GuillaumeDSM — APPROVED** [review]
great article !

File: 2022-08-what-is-future-trading.md (hunk 1 above)
Comment (Herklos) [reply 2022-08-what-is-future-trading.md]: Right 😄

**Pull request merged**